autoload/lsp/ui/vim/output.vim

```diff
@@ -0,0 +1,27 @@
+function! lsp#ui#vim#output#preview(content, ...) abort
```

Comment:
I would prefer to use a dictionary instead of variable args if possible `...`

Reply:
Done